src/app/(post-login)/submission/program/[shortName]/clinical-submission/data.tsx

```diff
@@ -39,10 +39,12 @@ const CLINICAL_FILE_ORDER = [
 
 // parse out nulls, undefined, provide sensible defaults so type checking isnt a scattered nightmare
 export const parseGQLResp: (v: any) => ClinicalSubmission = (gqlData) => {
-	const clinicalState = gqlData?.clinicalSubmissions.state;
-	const clinicalVersion = gqlData?.clinicalSubmissions.version || '';
+	const clinicalSubmissions = gqlData?.clinicalSubmissions;
 
-	const fileErrors = gqlData?.clinicalSubmissions.fileErrors;
+	const clinicalState = clinicalSubmissions.state;
+	const clinicalVersion = clinicalSubmissions.version || '';
+
+	const fileErrors = clinicalSubmissions.fileErrors;
```

Comment:
this function is to parse the gql data and format it for the different components on the page.
I moved the data loading section to a parent component. now data will not be undefined.
cleans up this function a bit.